packages/client/src/lib/util.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { formatPhoneNumber } from '../../../common/util'
+
+describe('formatPhoneNumber', () => {
+  const validPhones = [
+    {
```

Comment:
Match only `^\+1\d{10}(x\d+)?$` and make it look like '(123) 456-7899 Ext 240'.  Otherwise, just pass it through.